client/sse.js

```diff
@@ -0,0 +1,74 @@
+import createDebug from 'debug';
+import { EventEmitter } from 'events';
+
+const debug = createDebug('sse-client');
+
+const Events = {
+  open: '#open'
```

Comment:
Native [`open`](https://developer.mozilla.org/en-US/docs/Web/API/EventSource/open_event) event is _shadowed_ by server sent `#open` in order to retrieve server-generated connection id.